src/Text/Pandoc/Shared.hs

```diff
@@ -665,8 +666,12 @@ taskListItemToAscii = handleTaskListItem toMd
 handleTaskListItem :: ([Inline] -> [Inline]) -> Extensions -> [Block] -> [Block]
 handleTaskListItem handleInlines exts bls =
   if Ext_task_lists `extensionEnabled` exts
-  then handleItem bls
+  then handleTaskListItemNoExtsCheck handleInlines bls
   else bls
+
+handleTaskListItemNoExtsCheck :: ([Inline] -> [Inline]) -> [Block] -> [Block]
+handleTaskListItemNoExtsCheck handleInlines bls =
+  handleItem bls
   where
     handleItem (Plain is : bs) = Plain (handleInlines is) : bs
     handleItem (Para is  : bs) = Para  (handleInlines is) : bs
```

Comment:
I'd like to avoid the API change this would require.
Why not just call `handleTaskListItem`, and add `Ext_task_lists` to the extensions parameter when calling it?

Reply:
done